src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
Read about [interface naming convention ](https://www.baeldung.com/java-interface-naming-conventions)
```suggestion
public interface AreaCalculator {
```